docs/basics/networkobject.md

```diff
@@ -21,9 +21,7 @@ When spawning a NetworkObject, the `NetworkObject.GlobalObjectIdHash` value init
 You can use [NetworkBehaviours](networkbehaviour.md) to add your own custom Netcode logic to the associated NetworkObject.
 
 :::warning
-
-The order of networked objects matters. Make sure to load any NetworkBehaviour components before the Network Object component on the GameObject.
-
+The order of components on a networked GameObject matters. When adding netcode components to a GameObject, ensure that the NetworkObject component is ordered before any NetworkBehaviour components.
```

Comment:
@NoelStephensUnity Part of the user query was also about script execution order. Should we mention that in addition to component order? E.g.:

'Similarly, ensure that NetworkObject scripts execute before any NetworkBehaviour scripts.'

Reply:
There are no `NetworkObject` scripts but the order in which the `NetworkBehaviour` components are presented within the inspector view (i.e. starting after the `NetworkObject` component) is the order in which each associated `NetworkBehaviour.OnNetworkSpawn` method will be invoked. So, any properties that get set during `NetworkBehaviour.OnNetworkSpawn` will be set in the order each `NetworkBehaviour` component is set.

We might mention that users can avoid execution order issues in any `NetworkBehaviour` component scripts that have dependencies to other `NetworkBehaviour` components associated with the same `NetworkObject` by placing that/those script(s) snippet(s) within an overridden `NetworkBehaviour.OnNetworkPostSpawn` method. The `NetworkBehaviour.OnNetworkPostSpawn` method is invoked on each `NetworkBehaviour` component after all `NetworkBehaviour` components associated with the same `NetworkObject` component have had their `NetworkBehaviour.OnNetworkSpawn` methods invoked (but they will still be invoked in the same execution order defined by their relative position to the `NetworkObject` component when viewed within the Unity editor inspector view.

Reply:
Thanks for the additional detail Noel ^_^ I've upgraded this from an admonition box to a full on section, since I think this is useful information.